spacepy/toolbox/__init__.py

```diff
@@ -600,22 +600,28 @@ def human_sort( l ):
     return l
 
 
-def dictree(in_dict, verbose=False, spaces=None, levels=True, attrs=False, **kwargs):
+def dictree(in_dict, verbose=False, spaces=None, levels=True, attrs=False,
+            print_out=True, **kwargs):
     """
     pretty print a dictionary tree
 
     Parameters
     ----------
     in_dict : dict
         a complex dictionary (with substructures)
-    verbose : boolean (optional)
+    verbose : bool, default True
```

Comment:
verbosity default is False

Reply:
Well that's embarrassing. I'm guessing a copy/paste failure. Fixed in rebase.